src/office-toolbox.ts

```diff
@@ -231,8 +231,14 @@ async function sideload(application: string, manifestPath: string) {
       application = await promptForApplication();
     }
 
-    manifestPath = await checkAndPromptForPath(application, manifestPath);
-    await util.sideload(application, manifestPath);
+    if(application === 'word' || application === 'excel' || application === 'powerpoint'){
```

Comment:
Let's make this more data-driven by adding an applicationProperty "canSideload": true/false and use that here. It can then just check `if (applicationProperties[application].canSideload) {`